apis/v1alpha2/httproute_types.go

```diff
@@ -530,6 +530,23 @@ const (
 	HTTPRouteFilterExtensionRef HTTPRouteFilterType = "ExtensionRef"
 )
 
+// HTTPHeader represents an HTTP Header name and value as defined by RFC 7230.
+type HTTPHeader struct {
+	// Name is the name of the HTTP query param to be matched. This must be an
+	// exact string match. (See
```

Comment:
Why is this an exact string match? Below we mention that header names are case insensitive. I think you may have meant to refer to a different RFC: https://datatracker.ietf.org/doc/html/rfc2616#section-4.2.

Reply:
I assume that an exact header match is case-insensitive?

Reply:
Exact/Regex is how we match the value, the header name has always been exact (with the exception being case sensitivity).